index.js

```diff
@@ -0,0 +1,54 @@
+// Main Validate Password Function
+const validatePassword = password => {
+  let isMinLength = checkMinLength(password)
+  let isLowerCase = false
+  let isUpperCase = false
+  let isNumeric = false
+  let isSpecial = false
+
+  for (let i = 0; i < password.length; i++) {
+    !isLowerCase ? isLowerCase = checkLowerCase(password[i]) : isLowerCase = true
+    !isUpperCase ? isUpperCase = checkUpperCase(password[i]) : isUpperCase = true
+    !isNumeric ? isNumeric = checkNumeric(password[i]) : isNumeric = true
+    !isSpecial ? isSpecial = checkSpecial(password[i]) : isSpecial = true
+  }
+
+  return (isMinLength && isLowerCase && isUpperCase && isNumeric && isSpecial)
+}
+
+// Criteria Functions
+// Min Length is 8
+const checkMinLength = password => password.length >= 8
+
+// One character is lower case
+const checkLowerCase = chr => {
+  return (chr === chr.toLowerCase() && chr !== chr.toUpperCase())
```

Comment:
chr !== chr.toUpperCase()    Am guessing you are using this for number and special chars right because for them lower and upper case is the same.

i would have prefer to perhaps use else if statements but this is fine too